recipe/meta.yaml

```
@@ -1,5 +1,6 @@
 {% set name = "libestools" %}
 {% set version = "2.5.1" %}
+{% set ver2 = '.'.join(version.split('.')[0:2]) %}
 
 package:
   name: {{ name|lower }}
```

```suggestion
  name: {{ name|lower }}-split
```
Toplevel should be named libestools-split otherwise, libestools tests are skipped.